### DISCUSSION TITLE
[RESOLVED] Where's the Debian Testing change log?

### DISCUSSION THREAD

**Discussion by arsinclair** [doublecmd/doublecmd#1781] [General]
I'm on Debian Testing and I just received the info that Double Commander could be upgraded from version "1.1.16-0+svn52" to version "1.1.16-1".

Where is the change log? How can I identify what's changed between those versions?

**Comment by JonnyTech** [comment]
Your package maintainer will know which arbitrary points in the history they refer to as "1.1.16-0+svn52" and "1.1.16-1" (we cannot guess!) but the full commit history is here:

https://github.com/doublecmd/doublecmd/commits/master/

**Reply by arsinclair** [reply]
It's maintained by Alex2000, they have their own package registry.

**Comment by alexx2000** [comment]
Nothing changed, package with version "1.1.16-0+svn520" from my repo, while package with version "1.1.16-1" from Debian Testing repo.

**Reply by arsinclair** [reply]
Oh, so it was uploaded to Debian Testing and I can remove the custom registry from the apt sources?

**Reply by alexx2000** [reply]
> it was uploaded to Debian Testing

Yes

> I can remove the custom registry from the apt sources?

No, if you want to get updates quickly. Version 1.1.17 is almost ready.

**Reply by arsinclair** [reply]
Alright, thanks a lot for the quick response!
